content/en/real_user_monitoring/reactnative/mobile_vitals.md

```diff
@@ -28,6 +28,8 @@ To access the RUM mobile app performance dashboard, scroll down to the Monitor P
 
 Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph.
```

Comment:
Can we add a section similar to the one we have for Flutter to advertise for the JS refresh rate?

Something like so:

```suggestion
Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph.

When viewing performance details by the **View** event type, the following event timing and mobile vitals information is available:

- UI Refresh Rate Average
- JS Refresh Rate Average
- CPU Tickets Per Second
- Memory Average
```

Reply:
Hi @louiszawadzki, based on @timmld's feedback, I will actually remove this list in favor of a table similar to the one here: https://docs.datadoghq.com/real_user_monitoring/android/mobile_vitals/#metrics. I could use your help in defining these metrics!

Reply:
Awesome! For CPU ticks and memory usage you can reuse the definition from your link.

We also have **Slow renders** which are linked to Refresh Rates but less important, so I would suggest to update it and put it after the refresh rates whose definition could be:

**Refresh Rate Average**
To ensure a smooth, [jank-free](http://jankfree.org/) user experience, your application should render frames in under 60Hz.

RUM tracks the application’s [main thread display refresh rate](https://reactnative.dev/docs/performance#what-you-need-to-know-about-frames) using @view.refresh_rate_average and @view.refresh_rate_min view attributes.

Note: Refresh rates are normalized on a range of zero to 60fps. For example, if your application runs at 100fps on a device capable of rendering 120fps, Datadog reports 50fps in Mobile Vitals.

**JS Refresh Rate Average**
To ensure a smooth, [jank-free](http://jankfree.org/) user experience, your application should render frames in under 60Hz.

RUM tracks the application’s [javascript thread display refresh rate](https://reactnative.dev/docs/performance#what-you-need-to-know-about-frames) using @view.js_refresh_rate.average, @view.js_refresh_rate.min, @view.js_refresh_rate.max view attributes.

Note: Refresh rates are normalized on a range of zero to 60fps. For example, if your application runs at 100fps on a device capable of rendering 120fps, Datadog reports 50fps in Mobile Vitals.

**Slow renders**
To ensure a smooth, [jank-free](http://jankfree.org/) user experience, your application should render frames in under 60Hz.

With slow rendering, you can monitor which views have an average frame rate under 55fps.

Note: Refresh rates are normalized on a range of zero to 60fps. For example, if your application runs at 100fps on a device capable of rendering 120fps, Datadog reports 50fps in Mobile Vitals.